src/lib.rs

```diff
@@ -525,14 +525,14 @@ fn prompt_postgres_source(config: &mut Configuration) -> Result<()> {
         .allow_empty(false)
         .interact_text()
         .unwrap();
-    let schema: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("Schema:")
-        .default("public".to_string())
+    let origin: String = Input::with_theme(&ColorfulTheme::default())
+        .with_prompt("Origin:")
+        .default("*".to_string())
         .allow_empty(false)
         .interact_text()
         .unwrap();
-    let tables: String = Input::with_theme(&ColorfulTheme::default())
-        .with_prompt("Tables:")
+    let query: String = Input::with_theme(&ColorfulTheme::default())
+        .with_prompt("Query:")
         .default("*".to_string())
```

Comment:
Does it make sense to update default values to some query which will actually work? like `select 1` or smth?